pandas/core/generic.py

```diff
@@ -2967,11 +2967,33 @@ def add_prefix(self, prefix):
 
         Parameters
         ----------
-        prefix : string
+        prefix : str
+            The string to add before each item name.
 
         Returns
         -------
-        with_prefix : type of caller
+        Series
+            Original Series with updated item names.
+
+        See Also
+        --------
+        pandas.Series.add_suffix: Add a suffix string to panel items names.
+
+        Examples
+        --------
+        >>> s = pd.Series([1,2,3,4])
```

Comment:
Missing spaces after commas to pass PEP-8

Reply:
Interesting, the PEP-8 passed with any comments, it could be updated with this case?